pkgs/os-specific/linux/firejail/default.nix

```
@@ -25,6 +25,11 @@ stdenv.mkDerivation {
     substituteInPlace etc/firejail.config --replace \
       '# follow-symlink-as-user yes' \
       'follow-symlink-as-user no'
+
+    # Fix the path to 'xdg-dbus-proxy' hardcoded in the 'common.h' file
+    substituteInPlace src/include/common.h --replace \
+      '/usr/bin/xdg-dbus-proxy' \
+      '${xdg-dbus-proxy}/bin/xdg-dbus-proxy'
```

```suggestion
    substituteInPlace src/include/common.h \
      --replace '/usr/bin/xdg-dbus-proxy' '${xdg-dbus-proxy}/bin/xdg-dbus-proxy'
```

Just a minor nitpick. Otherwise LGTM.

---

Thanks, fixed!